parts/linux/cloud-init/artifacts/cse_cmd.sh

```diff
@@ -1,8 +1,17 @@
 PROVISION_OUTPUT="/var/log/azure/cluster-provision-cse-output.log";
 echo $(date),$(hostname) > ${PROVISION_OUTPUT};
 {{if ShouldEnableCustomData}}
-/bin/bash -c "source /opt/azure/containers/cloud-init-status-check.sh; handleCloudInitStatus \"${PROVISION_OUTPUT}\"; returnStatus=\$?; echo \"Cloud init status check exit code: \$returnStatus\" >> ${PROVISION_OUTPUT}; exit \$returnStatus" >> ${PROVISION_OUTPUT} 2>&1;
-cloudInitExitCode=$?;
+CLOUD_INIT_STATUS_SCRIPT="/opt/azure/containers/cloud-init-status-check.sh";
+cloudInitExitCode=0;
+if [ -f "${CLOUD_INIT_STATUS_SCRIPT}" ]; then
+	/bin/bash -c "source ${CLOUD_INIT_STATUS_SCRIPT}; handleCloudInitStatus \"${PROVISION_OUTPUT}\"; returnStatus=\$?; echo \"Cloud init status check exit code: \$returnStatus\" >> ${PROVISION_OUTPUT}; exit \$returnStatus" >> ${PROVISION_OUTPUT} 2>&1;
+    cloudInitExitCode=$?;
+else
+    cloud-init status --wait > /dev/null 2>&1;
```

Comment:
maybe change this bit to something like the following?

```suggestion
    cloud-init status --wait > /dev/null 2>&1;
    cloudInitExitCode=$?
```

that way L15 would apply to both cases

Reply:
non-blocking